<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Index</title>
    
    <link rel="stylesheet" href="{{ url_for('static',filename='styles/styles.css') }}"> 
    <link rel="stylesheet" href="{{ url_for('static',filename='styles/pygment_trac.css') }}"> 
    <meta name="viewport" content="width=device-width"> 
  </head>

    <style>
    .button {
      padding: 5px 10px;
      text-align: center;
      text-decoration: none;
      display: inline-block;
      font-size: 20px;
      margin: 1px 1px;
      cursor: pointer;
      background-color: white; 
      color: black; 
      border: 2px solid #4CAF50;
    }
    </style>

  <body>
    <div class="wrapper">
      <header>
        <h1>ADpred</h1>
        <p>Erijman et al., Nature-cell-Science, 2020</p>
        <p class="view"><a href="https://github.com/aerijman/TADs-analysis">View the Project on GitHub <small>aerijman/ADpred</small></a></p>
        <ul>
          <li><a href="https://github.com/aerijman/TADs-analysis">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/aerijman/TADs-analysis">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/aerijman/TADs-analysis">Fork On <strong>GitHub</strong></a></li>
        </ul>
    
            <form action="{{ url_for('index') }}" method="POST" style="width: 100%; float:left"> 
                    <div style="width: 50%; float:left">
                        <!--Paste amino-acid sequence / uniprot ID / common name:--><br>
                        <textarea name="sequence" rows="15" cols=32" placeholder= "Paste sequence or protein identifier here.&#10;Fasta or raw sequence are always faster." style="font-size: 15px;"; required></textarea><br>
                        <!--<input type="submit" value="predict">-->
                        <button class="button">Predict</button>
                    </div>
            </form>
            <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
            <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js"></script>
            <div class="chart" id="bargraph" style="position:fixed; bottom:1;left:2;width:400px;"> 
                <script>
                    var graphs = {{plot | safe}};
                    Plotly.plot('bargraph',graphs,{});
                </script>
            </div>               


      </header>

      <section>
        <h1>What are Activation Domains</h1>

        <p><em>View the <a href="http://github.github.com/github-flavored-markdown/sample_content.html">source of this content</a>.</em></p>

        <p>Transcription activators are regulatory factors that stimulate transcription in response to signaling pathways controlling processes such as development, growth and stress response. Misregulation of activators or mutations within them can lead to many human diseases and syndromes. Each activator contains one or more transcription activation domains (ADs) that usually target coactivators – complexes that contact the basal transcription machinery and/or have chromatin modifying activity. AD-coactivator binding initiates a series of events leading to productive transcription initiation, elongation and reinitiation. There are hundreds of cellular activators with distinct ADs, but most target only a small number of coactivators such as Mediator, TFIID, Swi/Snf, SAGA, NuA4 and p300. Broadly acting ADs can target several of these coactivators, allowing them to act on a large set of genes with different coactivator requirements. ADs have also been implicated in promoting the formation of intracellular condensates at enhancers, triggering the recruitment of a large dynamic network of coactivators and other factors responsible for gene activation.</p>
        <p>Nearly all characterized eukaryotic ADs are contained within intrinsically disordered protein regions (IDRs) that lack a stable 3D structure in the absence of a binding partner. In many systems apart from transcription, molecular recognition by IDRs is mediated by short linear motifs (SLMs), 3-10 residue sequence motifs found in otherwise unrelated proteins. In contrast, AD function is encoded by a wide range of seemingly unrelated sequences. For example, by characterizing natural ADs it was observed that, while AD sequences can be moderately conserved in closely related orthologs (Pacheco et al., 2018), no common sequence motif has been found when comparing ADs from different transcription factors. In addition, small-scale screens for ADs using random sequences of varying length fused to a DNA binding domain found that ~1% of these sequences encoded AD function. Finally, prior work analyzing natural and synthetic ADs have found several sequence features that correlate with AD function including intrinsic disorder, the presence of acidic, hydrophobic, and aromatic residues, low sequence complexity, net negative charge (or lack of positive charge) and, in some cases, alpha helix propensity.</p>
        <p>Recent structural work and molecular analysis suggests that a large class of ADs recognize coactivators via a dynamic “fuzzy” protein-protein interface. For example, the yeast activator Gcn4 contains tandem ADs that bind four structured domains in the Mediator subunit Med15. Structural analysis showed that the individual AD-Med15 interactions are dynamic, and that the two factors appear to interact via a cloud of hydrophobicity rather than through sequence-specific interactions. This binding mechanism does not require a unique sequence motif for AD function. Because of this, it has been difficult to predict sequences with AD function and to understand which features promote their dynamic binding properties and specificity. Understanding these fundamental properties of ADs is essential toward progress in determining the molecular basis of AD specificity for certain coactivators, dissecting mechanisms used in gene activation, and in predicting the consequences of naturally occurring mutations on AD function.</p>

        <h2>Why is important to recognize ADs within protein sequences</h2>

        <p>TODO</p>

       <!-- <ul>
          <li>SHA: be6a8cc1c1ecfe9489fb51e4869af15a13fc2cd2</li>
          <li>User@SHA ref: <a href="https://github.com/mojombo/product/commit/be6a8cc1c1ecfe9489fb51e4869af15a13fc2cd2" class="commit-link">mojombo@<tt>be6a8cc</tt></a></li>
          <li>User/Project@SHA: <a href="https://github.com/mojombo/god/commit/be6a8cc1c1ecfe9489fb51e4869af15a13fc2cd2" class="commit-link">mojombo/god@<tt>be6a8cc</tt></a></li>
          <li>#Num: <a href="https://github.com/github/product/issues/1" class="issue-link" title="Baseline: What is up in the air right now?">#1</a></li>
          <li>User/#Num: mojombo#1</li>
          <li>User/Project#Num: <a href="https://github.com/mojombo/god/issues/1" class="issue-link" title="The server is not available (or you do not have permissions to access it)">mojombo/god#1</a></li>
        </ul> --> 
        <pre><code>def robot_invasion
  puts("robot " * 1000)
end
</code></pre>

        <p>You see, that was formatted as code because it's been indented by four spaces.</p>

        <p>How about we throw some angle braces and ampersands in there?</p>

        <pre><code>&lt;div class="footer"&gt;
    &amp;copy; 2004 Foo Corporation
&lt;/div&gt;
</code></pre>

        <h2>Set in stone</h2>

        <p>Preformatted blocks are useful for ASCII art:</p>

        <pre>             ,-.
    ,     ,-.   ,-.
   / \   (   )-(   )
   \ |  ,.&gt;-(   )-&lt;
    \|,' (   )-(   )
     Y ___`-'   `-'
     |/__/   `-'
     |
     |
     |    -hrr-
  ___|_____________
        </pre>

        <!-- <blockquote>
        <p>I wish you'd have given me this written question ahead of time so I<br>
        could plan for it... I'm sure something will pop into my head here in<br>
        the midst of this press conference, with all the pressure of trying to<br>
        come up with answer, but it hadn't yet...</p>

        <p>I don't want to sound like<br>
        I have made no mistakes. I'm confident I have. I just haven't - you<br>
        just put me under the spot here, and maybe I'm not as quick on my feet<br>
        as I should be in coming up with one.</p>
        </blockquote>

        <table>
          <tbody>
        <tr>
            <th>ID</th>
        <th>Name</th>
        <th>Rank</th>
          </tr>
          <tr>
            <td>1</td>
        <td>Tom Preston-Werner</td>
        <td>Awesome</td>
          </tr>
          <tr>
            <td>2</td>
        <td>Albert Einstein</td>
        <td>Nearly as awesome</td>
          </tr>
        </tbody>
        </table> --> 
    </section> 
      <footer>
        <p>This project is maintained by <a href="https://github.com/aerijman/TADs-analysis">Ariel Erijman</a></p>
        <p><small>The theme is basaed on  &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script type=text/javascript src="{{ url_for('static',filename='javascripts/scale.fix.js') }}"></script>
  </body>
</html>
